release-notes/DxMs/cloudgateway_change_log.md

```diff
@@ -4,6 +4,16 @@ uid: cloudgateway_change_log
 
 # Cloud Gateway change log
 
+#### 16 July 2024 - Fix - CloudGateway 2.13.14 - Requests could get no response in an edge case leading to timeouts [ID_40208]
+
+If an issue occurred while handling a remote access web API call, it could happen that no response was returned, which would lead to the call going into timeout. Now the CloudGateway will try to return an internal server response immediately.
```

Comment:
We always write it in one piece, the name of the DxM

Reply:
In that case, I'd add "DxM" to make clear that it's the exact name of the DxM. The spelling without space looks rather odd, so when just referring to the module in general, I prefer to add it.
```suggestion
If an issue occurred while handling a remote access web API call, it could happen that no response was returned, which would lead to the call going into timeout. Now the CloudGateway DxM will try to return an internal server response immediately.
```